requirements.txt

```diff
@@ -14,6 +14,7 @@ six~=1.13
 tenacity~=6.2
 typing-extensions~=4.10
 packaging~=23.1
+pylint
```

Comment:
Please remove this too. Pylint in a bad shape now so it's not even added to dev dependencies. To run locally, use https://github.com/grpc/psm-interop/blob/main/bin/pylint.sh for now. I'll get it updated some time soon.